### ISSUE TITLE
Use fragments instead of new activities

### ISSUE THREAD

**Issue by mgrddsj** [nextcloud/android#5480]
### Steps to reproduce
1. Open navigation drawer
2. Opening "Activities", "Notifications", "Auto Upload", "Uploads", and "Deleted files
3. 

### Expected behaviour
Opening the feature in the same activity (com.owncloud.android.ui.activity.FileDisplayActivity), just switch the fragment, like opening "Favorites", "Photos", etc. 

### Actual behaviour
Opens the feature in a new activity, and shows an animation of opening a new activity. 

### Environment data
Android version:
Android 9

Device model: 
OnePlus 5 (ONEPLUS A5000)

Stock or customized system:
Stock-like H2OS 9.0.7
(Preinstalled on Chinese models of OnePlus Phones, similar to Oxygen OS) 

Nextcloud app version:
3.10.1 from Play Store

Nextcloud server version:
17.0.3

#### Web server error log
(Not server related problem.) 

#### Nextcloud log (data/nextcloud.log)
(App not crashed, just an user experience enhancement.)

**Comment by tobiasKaminsky** [comment]
Thanks for your suggestion, but this is not a bug.
We decided it "back then" to have a separate activity, to not run into problems as FileDisplayActivity would be then always the one big activity.
But sure, there is room for improvement.

If you want to help us, please first describe what and how you want to do it, as this touches central parts of our app.

**Comment by mario** [comment]
I kindof agree with @mgrddsj, but I also agree with @tobiasKaminsky that this is an architectural change/improvement (and a rather large one at that) rather than a bug.

@tobiasKaminsky activity is a simple container - no need for it to be big.

**Comment by mgrddsj** [comment]
Yes, I knew it is an enhancement, I just didn't find a way to remove the bug tag. 
It is indeed a large architectural change, and I'm still learning Android development, so I guess I can't help with this issue. 

I also advise to change the animation of opening the new activity to fade as a temporary fix, so that the visual experience is relatively better.

**Comment by tobiasKaminsky** [comment]
> I also advise to change the animation of opening the new activity to fade as a temporary fix, so that the visual experience is relatively better.

This is a nice quick win, good idea :+1:

**Comment by tobiasKaminsky** [comment]
> @tobiasKaminsky activity is a simple container - no need for it to be big.

True, but currently FDA is very big…so we would have to change this.
